lib/suspenders/app_builder.rb

```diff
@@ -401,7 +401,7 @@ def setup_segment
 
     def setup_bundler_audit
       copy_file "bundler_audit.rake", "lib/tasks/bundler_audit.rake"
-      append_file "Rakefile", %{\ntask default: "bundler:audit"\n}
+      append_file "Rakefile", %{\ntask default: "bundle:audit"\n}
```

Comment:
Should this be `task default: ["spec", "bundle:audit"]`?

Reply:
Although it may be clearer what's going on defined as one line, is it lumping together responsibilities? I think we should augment the default task with `bundle:audit` independently of expecting the specs to run.

Having said, I did find Rake's behaviour of find the append (rather-than-redefine) mechanism confusing at first... it's not clear.

```ruby
# Rakefile

task :foo do
  puts "FOO"
end

task :bar do
  puts "BAR"
end

task(:default).clear
task default: :foo
task default: :bar
```

```sh
$ bundle exec rake
FOO
BAR
```

---

One way we could make this clearer is writing it as:

```
Rake::Task[:default].enhance ["bundle:audit"]
```

Then there's less confusion whether the task is being overwritten or modified.

Reply:
@croaky also this comment on the original PR for context: https://github.com/thoughtbot/suspenders/pull/831#issuecomment-285701717

Reply:
> Although it may be clearer what's going on defined as one line, is it lumping together responsibilities? I think we should augment the default task with bundle:audit independently of expecting the specs to run.

I think I'd expect `rake` in a Suspender-ized app to run `spec` then `bundle:audit` at this point. If that's what is happening via multiple `task default:` invocations, that's great.

Related: I learned about https://github.com/presidentbeef/brakeman last week. Seems like it covers security cases additional to Bundle Audit. Might be worth experimenting with on some apps.

Reply:
@croaky we are using brakeman in our suspenders fork. I can cherry-pick and create a PR if you are interested!